src/odemis/acq/millmng.py

```diff
@@ -323,7 +350,7 @@ def _mill_all_settings(self, site: CryoFeature):
             # X is horizontal axis, Y is vertical axis
             pxs = milling_settings.pixelSize.value
             probe_size = max(pxs)
-            overlap = [1 - (pxs[0] / probe_size), 1 - (pxs[1] / probe_size)]  # Always >= 0
+            overlap = [1 - (float(pxs[0]) / float(probe_size)), 1 - (float(pxs[1]) / float(probe_size))]  # Always >= 0
```

Comment:
What are you trying to do here with the `float()`? If pxs and probe_size are numbers, I don't think this will have any effect. Could this be that probe_size is a string? If you rebase against master you should get my change that should help catch such issue early.